### DISCUSSION TITLE
Additional check for row creation

### DISCUSSION THREAD

**Discussion by DuckMcQu4ck** [serenity-is/Serenity#6410] [Help]
It is possible to do check at creation of a row? For example, show similar row and ask for additional confirmation to save?

**Comment by edwardch** [comment]
Have a look at #6169 

If you want a popup instead of validation error, you can just do a confirm popup and show some details to the user to confirm.

**Reply by edwardch** [reply]
This [comment on How to show confirmation before Save?](https://github.com/serenity-is/Serenity/issues/814#issuecomment-363510483)  might also be useful

**Comment by minhhungit** [comment]
@DuckMcQu4ck I guess you can do it by overwrite https://github.com/serenity-is/Serenity/blob/b3fa3ddfc308aada4cfa93b6c555477d09ea6585/src/Serenity.Net.Services/RequestHandlers/Save/SaveRequestHandler.cs#L24
